package.json

```
@@ -8,7 +8,8 @@
     "eslint": "eslint src/**/*.ts --report-unused-disable-directives",
     "verify": "npm run tsc && npm run eslint & npm run webpack",
     "watch": "chokidar \"src/**/*.ts\" -i \"src/**/*.d.ts\" -c \"npm run webpack\"",
-    "webpack": "webpack"
+    "webpack": "webpack",
+    "build": "tsc -p ./src"
```

Is this intentional? There's no `tsconfig.json` in `./src`, so I'd think `npm run tsc` would be enough?

```suggestion
    "webpack": "webpack"
```